storage-manager/src/IOCoordinator.cpp

```diff
@@ -1001,10 +1048,13 @@ int IOCoordinator::copyFile(const char *_filename1, const char *_filename2)
         for (const auto &object : objects)
         {
             bf::path journalFile = journalPath/firstDir1/(object.key + ".journal");
-            // XXXPAT: Need to follow up on this.  this is the wrong length to use here, but changing it
-            // to use the right length is causing an error somewhere else.  Not sure why yet.
-            metadataObject newObj = meta2.addMetadataObject(filename2, object.length);
-            assert(newObj.offset == object.offset);
+            
+            // originalLength = the length of the object before journal entries.
+            // the length in the metadata is the length after journal entries
```

Comment:
The error mentioned in the old comment is that the new metadata entry would get the original size, not the merged size.  Fixed that in the next line if the original size != merged size.